cadquery/occ_impl/shapes.py

```diff
@@ -428,17 +430,16 @@ def exportStl(
             Default is 1e-3, which is a good starting point for a range of cases.
         :param angularTolerance: Angular deflection setting which limits the angle between subsequent segments in a polyline. Default is 0.1.
         :param ascii: Export the file as ASCII (True) or binary (False) STL format.  Default is binary.
+        :param isToleranceRelative: If True, deflection used for discretization of each edge will be <tolerance> * <size of="" edge>="">. Deflection used for the faces will be the maximum deflection of their edges.
+            Setting this value to True may be problematic for parts with a wide range of feature sizes, as large features will be very faceted, or small features will be very dense.
         """
-
-        mesh = BRepMesh_IncrementalMesh(self.wrapped, tolerance, True, angularTolerance)
-        mesh.Perform()
+        # The constructor used here automatically calls mesh.Perform(). https://dev.opencascade.org/doc/refman/html/class_b_rep_mesh___incremental_mesh.html#a3a383b3afe164161a3aa59a492180ac6
+        BRepMesh_IncrementalMesh(
+            self.wrapped, tolerance, isToleranceRelative, angularTolerance, isParallel
+        )
 
         writer = StlAPI_Writer()
-
-        if ascii:
-            writer.ASCIIMode = True
-        else:
-            writer.ASCIIMode = False
+        writer.ASCIIMode = bool(ascii)
```

Comment:
```suggestion
        writer.ASCIIMode = ascii
```

Reply:
Huh - didn't realize I could just commit it from the browser here. Change made, thanks for the feedback. Just wanted to be certain I didn't change the behavior at all.